From 06532be466d8f78d9318eac588f6c521136f4ec1 Mon Sep 17 00:00:00 2001
From: Linu Cherian <linu.cherian@marvell.com>
Date: Sun, 9 Dec 2018 23:34:48 +0530
Subject: [PATCH 0854/1051] soc: octeontx2: Fix module build error

EXPORT cgx_lmac_ptp_config

Change-Id: Icc90a642e090a571343830595c761cba242a94dc
Signed-off-by: Linu Cherian <linu.cherian@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/1761
Tested-by: sa_ip-sw-jenkins
Reviewed-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/soc/marvell/octeontx2/cgx.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/soc/marvell/octeontx2/cgx.c b/drivers/soc/marvell/octeontx2/cgx.c
index ef0766ab4463..2903daa10bb2 100644
--- a/drivers/soc/marvell/octeontx2/cgx.c
+++ b/drivers/soc/marvell/octeontx2/cgx.c
@@ -315,6 +315,7 @@ void cgx_lmac_ptp_config(void *cgxd, int lmac_id, bool enable)
 		cgx_write(cgx, lmac_id, CGXX_SMUX_RX_FRM_CTL, cfg);
 	}
 }
+EXPORT_SYMBOL(cgx_lmac_ptp_config);
 
 int cgx_get_rx_stats(void *cgxd, int lmac_id, int idx, u64 *rx_stat)
 {
-- 
2.17.1

